application/base_script.sh

```diff
@@ -0,0 +1,40 @@
+xcp_d \
+    --participant_label 01
+    -t rest \
+    --bids-filter-file filter.json \
+    --combineruns \
+    --cifti \
+    --nthreads 1 \
+    --omp-nthreads 1 \
+    --mem_gb 10 \
+    # --use-plugin ??? \  # No Nipype plugin
+    -v \
+    --input-type fmriprep \
+    --smoothing 6 \
+    --despike \
+    -p 36P \
+    # -c ??? \  # No custom confounds
+    --min_coverage 0.5 \  # Parcels must have 50% coverage to be retained
+    --min_time 100 \  # Runs must have 100 seconds of low-motion data to be processed
+    --dummy-scans auto \  # Infer the number of dummy scans from the fMRIPrep confounds file
+    --random-seed 0 \  # Random seed for the exact-time subsampling
+    # --disable-bandpass-filter \  # Bandpass filter will be used
+    --lower-bpf 0.01 \
+    --upper-bpf 0.08 \
+    --bpf-order 2 \
+    --motion-filter-type notch \
```

Comment:
Which frequency is notched out?

Reply:
The 12-18 BPM frequency band.

Reply:
I always thought that notch filters were just for a specific frequency, and that a range would be a band-reject filter. If this is the BOLD terminology then I am ok with it